ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -405,6 +408,59 @@ S3Terminate(WT_STORAGE_SOURCE *storage, WT_SESSION *session)
     return (0);
 }
 
+/*
+ * S3Flush --
+ *     Flush file to S3 Store using AWS PutObject.
+ */
+static int
+S3Flush(WT_STORAGE_SOURCE *storageSource, WT_SESSION *session, WT_FILE_SYSTEM *fileSystem,
+  const char *source, const char *object, const char *config)
+{
+    S3_FILE_SYSTEM *fs = (S3_FILE_SYSTEM *)fileSystem;
+    int ret = fs->connection->PutObject(fs->bucketName, object, source);
```

Comment:
We could instead do:

`return (fs->connection->PutObject(fs->bucketName, object, source));`